src/widgets/transformer-editor.jsx

```diff
@@ -659,15 +659,15 @@ var ShapeTypes = {
                 return false;
             }
 
-            var line1_0 = [points1[1], points1[0]];
-            var line1_2 = [points1[1], points1[2]];
-            var line2_0 = [points2[1], points2[0]];
-            var line2_2 = [points2[1], points2[2]];
+            var line1a = [points1[1], points1[0]];
```

Comment:
Yeah, I agree that's silly. I tried it with lint exception comments but I thought it looked marginally better this way.